app-admin/genromfs/genromfs-0.5.2-r1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
just do this inline, i.e., inplace EAPI bump, this ebuild is simple enough, I will also check it again